### PR TITLE
TMDM-9903 [DI] Support merging task in TDS taskoutput component  

### PR DESCRIPTION
**Pull request by htyin**

### CODE REVIEW

**Comment by build-talend** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/242/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/242/)

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/250/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/250/)

**Comment by build-talend** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/253/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/253/)

**Comment by build-talend** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/256/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/256/)

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/257/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/257/)

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/260/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/260/)

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/261/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/261/)

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/262/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/262/)

File: components-datastewardship/src/main/java/org/talend/components/datastewardship/runtime/writer/TdsTaskWriter.java [hunk 1]
```diff
@@ -65,27 +81,106 @@ public void write(Object datum) throws IOException {
...
-        handleResponse(statusCode, resourceToCreate, record);
+    }                               
+   
+    @Override
+    public Result close() {
+        if (taskObjs.size() > 0) {
+            try {
+                submit();
+            } catch (IOException e) {
+                e.printStackTrace();
```
Comment (glzhou) [inline components-datastewardship/src/main/java/org/talend/components/datastewardship/runtime/writer/TdsTaskWriter.java]: Exception should be emitted not swallowed.

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/263/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/263/)

**Pull request merged**